cogs/cmd_interface/leaderboard.py

```diff
@@ -3,20 +3,34 @@
 from discord.ext import commands
 from lib.embed.interactable_leaderboard_embed import RankingView, InteractableLeaderboardEmbed
 from utils.llc_datetime import LLCMonth
+from utils.asset import Assets
 
 class Leaderboard(commands.Cog):
     def __init__(self, client):
         self.client = client
 
     rank_group = app_commands.Group(name = "leaderboard", description = "Ranking Group")
-    @rank_group.command(name = "current", description = "Take a look at LLC's Hall of Fame")
-    async def _leaderboard_current(self, interaction: discord.Interaction):
-        await interaction.response.defer(thinking = True)
+
+    @rank_group.command(name="current", description="Take a look at LLC's Hall of Fame")
+    @app_commands.describe(by_user="User sorted by role")
+    @app_commands.choices(
+        by_user=list(
+            map(
+                lambda x: app_commands.Choice(name=x, value=x), 
+                ["Leetcoder of the Month", "LLClass Student", "Core Members", "Bot Developer"]
```

Comment:
Refactor mảng này lên trên đi để dùng lại cho cả dòng 48 nữa. Và tham khảo comment trước để thử dùng với lambda với for in, thay vì list(map(...)) which I don't think is pythonic.